Apps/Sandcastle/gallery/3D Models.html

```diff
@@ -43,14 +111,16 @@
     var hpr = new Cesium.HeadingPitchRoll(heading, pitch, roll);
     var orientation = Cesium.Transforms.headingPitchRollQuaternion(position, hpr);
 
-    var entity = viewer.entities.add({
+    entity = viewer.entities.add({
         name : url,
         position : position,
         orientation : orientation,
         model : {
             uri : url,
             minimumPixelSize : 128,
-            maximumScale : 20000
+            maximumScale : 20000,
+            blendAmount : viewModel.blendAmount,
+            blendColor : Cesium.Color.fromHsl(viewModel.blendColor, 1.0, 0.5, viewModel.blendAlpha)
```

Comment:
Do our users think of this as blending?  Or just as applying a color to a model like [Billboard.color](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Billboard.js#L592)?

What do you think of renaming this to `color`?  I'm not sure what to rename `blendAmount` to...is there anywhere else in the Cesium API to mirror?